.github/workflows/fill_station_action.yml

```diff
@@ -69,7 +69,7 @@ jobs:
           images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}-${{ matrix.arch }}
           tags: |
             type=raw,value=latest,enable={{is_default_branch}}
-            type=ref,event=branch
+            type=ref,event=branch,enable=${{ github.ref != 'refs/heads/main' }}
```

Comment:
Why do we need this addition? If the github ref is main (e.g. it is the default branch, then doesn't the previous line gets matched instead of this line. So isn't this redundant?

Reply:
The CI failure on merger #49 was because it matches both lines and updates two images, one with tag main, and one with tag latest. This should make it just latest and allow merge to complete successfully

Reply:
Cool, I see! Thank you!